openshift/release/kn.yaml

```diff
@@ -11,12 +11,11 @@ plugins:
       - module: knative.dev/kn-plugin-event
         moduleSource: github.com/openshift-knative/kn-plugin-event
         version: release-1.14
-  # - name: kn-plugin-func
-  #   module: knative.dev/func
-  #   pluginImportPath: knative.dev/func/plugin
-  #   version: v1.15.0
-  #   replace:
-  #     - module: knative.dev/func
-  #       moduleSource: github.com/openshift-knative/kn-plugin-func
-  #       version: serverless-1.34
-  
\ No newline at end of file
+  - name: kn-plugin-func
+    module: knative.dev/func
+    pluginImportPath: knative.dev/func/plugin
+    version: v1.15.0
```

Comment:
Shouldn't this be v1.15.1?

Reply:
If we want to be very precise. :)

Thanks, good catch!